src/kudu/consensus/consensus_peers.cc

```diff
@@ -491,7 +491,7 @@ void Peer::ProcessResponseError(const Status& status) {
   }
 #endif
 
-  LOG_WITH_PREFIX_UNLOCKED(WARNING) << "Couldn't send request to peer " << peer_pb_.permanent_uuid()
+  KLOG_EVERY_N_SECS(WARNING, 60) << LogPrefixUnlocked() << "Couldn't send request to peer " << peer_pb_.permanent_uuid()
```

Comment:
line format looks off.

Reply:
The logmessage issue with tp2 glog is a linking issue which happened because glog has undergone a recent API change. Nothing to do with this change, but the way to build tp2 will have to be different than what we are used to.